backend/src/main/java/gov/cdc/usds/simplereport/service/AuditService.java

```diff
@@ -75,6 +97,8 @@ public void logRestEvent(
     log.trace("Saving audit event for {}", requestId);
     HttpRequestDetails reqDetails = new HttpRequestDetails(request);
     ApiUser userInfo = _userService.getCurrentApiUserInContainedTransaction();
+    logEvent(
+        new ConsoleApiAuditEvent(requestId, reqDetails, responseCode, userInfo, org, patientLink));
```

Comment:
Nice - so this changes nothing about how we are interacting with the DB, so we shouldn't have the same troubles as yesterday. Good fix!

Reply:
yeah fingers crossed! there is no interaction what so ever with any db objects